packages/fiori/src/themes/DynamicPageHeaderActions.css

```diff
@@ -0,0 +1,43 @@
+.ui5-dynamic-page-header-actions-root {
+	position: relative;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	width: 100%;
+	.ui5-dynamic-page-header-action {
+		position: relative;
+		z-index: 1;
+		min-width: 1.5rem;
+		height: 1.5rem;
+		background-color: var(--sapObjectHeader_Background);
```

Comment:
Currently, on the Quartz theme, the pin button icon vanishes and does not have the correct `background-color` when clicked. The same is valid when we interact with the other action button. When the expand button has the attribute `active` its icon and `background-color` differ from the design specification. Please, refer to [this](https://ui5.sap.com/resources/sap/ui/documentation/sdk/index.html?sap-ui-xx-sample-id=sap.f.sample.DynamicPageFreeStyle&sap-ui-xx-sample-lib=sap.f&sap-ui-xx-sample-origin=.&sap-ui-xx-dk-origin=https%3A%2F%2Fui5.sap.com&sap-ui-theme=sap_fiori_3_dark&sap-ui-rtl=false&sap-ui-density=sapUiSizeCompact) example from SAPUI5.

Reply:
Resolved with the following #8763